core/src/main/java/org/apache/carbondata/core/constants/CarbonV3DataFormatConstants.java

```diff
@@ -63,6 +63,6 @@
   /**
    * number of rows per blocklet column page default value
    */
-  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT = 32000;
+  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT = 10240;
```

Comment:
are we going to fix page size to 10240??

Reply:
no, just for testing, going to be removed